scenarios/centos-9/ci.yml

```diff
@@ -5,11 +5,25 @@ cifmw_installyamls_repos: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-o
 cifmw_use_libvirt: true
 cifmw_use_crc: true
 
+pre_infra:
+  - name: Download needed tools
+    inventory: "{{ cifmw_installyamls_repos }}/devsetup/hosts"
+    type: playbook
+    source: "{{ cifmw_installyamls_repos }}/devsetup/download_tools.yaml"
+
 pre_deploy:
   - name: Deploy toy ceph
     type: playbook
     source: ceph-deploy.yml
 
+post_ctlplane_deploy:
```

Comment:
I don't think this is a valid key for hooks? Or am I lagging with the code state?

Reply:
That's something that @raukadah added to tune rabbit, as he needed reduce the assigned resources of the deployed rabbit. Honestly, I didn't like the idea of having that single hook there, and we agreeded to refactor it a little bit afterwards, I have the removal of this hook and properly create a post/pre one for the ctrlplane it in my TODO list. I'd probably add a formal step playbook for the ctrlplane deployment, instead of having a single one for ctrlplane and edpm, as we may want to perform something after the controlplane gets deployed.